src/hooks/useActiveProject.tsx

```diff
@@ -36,40 +65,63 @@ export const ActiveProjectContextProvider = ({
   const [hookReturnValue, setHookReturnValue] = useState<ActiveProjectProps>(
     {},
   );
+  const { useGetItem, setItem } = useAsyncStorage();
```

Comment:
Why does this hook return a hook? why cant we do something like

```
const [storedProjectIdResult, setStoredProjectId] = useAsyncStorage(selectedProjectIdKey);
```

Reply:
We can and we should, I like that even better.